Modules/Core/Common/include/itkSymmetricSecondRankTensor.h

```diff
@@ -104,7 +104,11 @@ class ITK_TEMPLATE_EXPORT SymmetricSecondRankTensor : public FixedArray<TCompone
 
   /** Default-constructor.
    * \note The other five "special member functions" are defaulted implicitly, following the C++ "Rule of Zero". */
+#ifdef itkSymmetricSecondRankTensor_h // TODO Must be: #ifdef ITK_FUTURE_LEGACY_REMOVE
```

Comment:
Note: the initial CI run is expected to fail, because of the incorrect `#ifdef`. I just want to see if defaulting `SymmetricSecondRankTensor()` would cause any test failures, other than the GTest `CommonTypeTraits.SymmetricSecondRankTensorIsNotPOD`.